oak_functions/lookup/src/lib.rs

```diff
@@ -154,7 +154,7 @@ pub struct LookupDataManager<L: OakLogger + Clone> {
 
 #[derive(Debug, PartialEq)]
 pub enum UpdateStatus {
-    Started,
+    Extended,
```

Comment:
nit: "Extended" does not feel like an accurate description of the state. I can't really think of a better name, but perhaps "Extensible" or "Continuing" is more accurate (although none of them feel great).

Reply:
I agree, my reasoning: with `Extended` you have at least the relationship to `extend`. 

I'd leave it for now, because I am thinking of refactoring further by replacing the `UpdateActions` with 3 RPCs: `Extend`, `Finish`, and `Abort`. I believe that would make the `UpdateStatus` superflous as we have `ExtendResponse`.